File: AngularLovelyjubbly/ClientApp/src/app/shared/models/recordcategory.ts
export interface IRecordCategory {
    recordCategoryId: number;
    recordCategoryName: string;
    isPerSeason: boolean;
}

export class RecordCategory implements IRecordCategory {

    constructor(public recordCategoryId: number, public recordCategoryName: string, public isPerSeason: boolean) {
    }
}

File: AngularLovelyjubbly.Data.Sql.Models/RecordCategory.cs
namespace AngularLovelyjubbly.Data.Sql.Models
{
    public partial class RecordCategory
    {
        public RecordCategory()
        {
        }

        public int RecordCategoryId { get; set; }
        public string RecordCategoryName { get; set; }
        public bool IsPerSeason { get; set; }
    }
}
